pkgs/development/python-modules/dm-sonnet/default.nix

```diff
@@ -50,5 +50,6 @@ buildPythonPackage rec {
     homepage = "https://github.com/deepmind/sonnet";
     license = licenses.asl20;
     maintainers = with maintainers; [ onny ];
+    broken = true; # at 2022-09-23
```

Comment:
Please add a comment so that we know about the dependency issue.

Reply:
`etils` builds. I don't know why it errors like that. relaxDeps also did not solve problem.

Reply:
Comment added.